blocks/footer/__item/footer__item.css

```diff
@@ -0,0 +1,26 @@
+.footer__item {
+  margin-bottom: 10px;
```

Comment:
It would be better to put margin: 0 here. It'll look more like the element from the brief.